service/models.py

```diff
@@ -111,6 +111,14 @@ def update(self):
             raise DataValidationError("Update called with empty ID field")
         db.session.commit()
 
+    def suspend(self):
```

Comment:
@JustinChiangkc I don't think you should add this suspend() function to the model, because it does the exact same thing as the update() function. Just use the update function instead.